cookbook/configuration/apache_router.rst

```diff
@@ -61,20 +62,20 @@ To test that it's working, create a very basic route for the AppBundle:
         # app/config/routing.yml
         hello:
             path: /hello/{name}
-            defaults: { _controller: AppBundle:Demo:hello }
+            defaults: { _controller: AppBundle:Default:hello }
```

Comment:
I'm against advertising the usage of a DefaultController, as default is the most generic world in the universe. What about `AppBundle:Greet:hello` (it's an example anyways)?